cleverhans/attack_bundling.py

```diff
@@ -410,9 +410,10 @@ def bundle_attacks_with_goal(sess, model, x, y, adv_x, attack_configs,
                         run_counts,
                         goal, report, report_path,
                         attack_batch_size=attack_batch_size)
-  # Save after finishing each goal.
+  # Save after finishing all goals.
   # The incremental saves run on a timer. This save is needed so that the last
   # few attacks after the timer don't get discarded
+  report.completed = True
```

Comment:
The update to the bool needs to be before the save, so that the bool gets saved. The main time we're interested in the value of the bool is when we read a report back from disk, e.g. in print_report.py in the other PR.

Reply:
I see, that's the behavior added in #887  

Thanks for clarifying